policies/templates/gcp_network_enable_flow_logs_v1.yaml

```diff
@@ -59,8 +59,12 @@ spec:
             	asset.asset_type == "compute.googleapis.com/Subnetwork"
             
             	network := asset.resource.data
-            	enable_flow_logs := lib.get_default(network, "enableFlowLogs", false)
-            	enable_flow_logs == false
+            	legacy_enable_flow_logs := lib.get_default(network, "enableFlowLogs", false)
+            	log_config := lib.get_default(network, "logConfig", {})
+            	log_config_enable_flow_logs := lib.get_default(log_config, "enable", false)
```

Comment:
Based on the provider [documentation](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_subnetwork#log_config) I believe that the presence of this block denotes that logging is enabled. I don't think there is an `enable` argument in its schema to begin with.

I believe just the presence of this block can be used to assert that logs are enabled.

Reply:
Hello @g-awmalik , so, this "enable" field is actually a calculated field, it's generated through the [CAI conversion proccess](https://github.com/GoogleCloudPlatform/terraform-validator/blob/main/converters/google/resources/compute_subnetwork.go#L261).

So, when we verify whether it's true, we are checking if `lib.get_default(network, "logConfig", {})` didn't returned an empty map.

Reply:
Ok thanks for clarifying.